set.mm

```diff
@@ -144860,9 +144859,10 @@ Proper unordered pairs and triples (sets of size 2 and 3)
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
-  $( TODO-AV1: no lemma, but self-reliant theorem!  Lemma for ~ brfi1ind :  The
-     size of a set is the size of this set with one element removed, increased
-     by 1.  (Contributed by Alexander van der Vekens, 7-Jan-2018.) $)
+  $( The size of a finite set minus one of its elements is equal to the size of
+     that set minus one.  TODO-AV1: find a better label, since not only a lemma
+     but interesting in its own right.  (Contributed by Alexander van der
+     Vekens, 7-Jan-2018.) $)
   brfi1indlem $p |- ( ( V e. W /\ N e. V /\ Y e. NN0 )
```

Comment:
maybe hashdifsnp1? hashdifsn already exists ( `` ... -> ( # ` ( A \ { B } ) ) = ( ( # ` A ) - 1 )``), and "p1" stands for "plus 1" (i.e., for ` ( Y + 1 ) `).

Reply:
And it has the same length, so no rewrap needed !